mozilla/foundation.mozilla.org
# Integrate the pni-q3-2021 model updates into main

Closes #7528

This copies the PNI model changes over, with a single migration, with the associated cuts to tests/factories/etc for the "contact this company" information (phone number, email, twitter, live chat).

# Testing

- check out this branch
- run `inv clone-stage-db` so that you're running a db with real data
- run `inv test` to confirm all out tests still pass
- run `docker-compose up` and go the PNI url
- the model changes should lead to _the same presentation_ as is currently live on staging and product, because no template changes were copied over
- however, some things that should have changed: `product update` entries should now respect their CMS sort order
- the section on contacting the company is gone (because we removed those fields, so this PR also makes sure to remove the associated template code)
- you know what you worked on: please verify that all those things do what you remember they should do, even if the templates won't show that =)
